index.js

```diff
@@ -59,14 +58,46 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-    if (element.style.color === "green") {
-        element.style.color = "black"
+
+header.addEventListener("click", function () {
+    if (header.style.color !== 'red') {
+        header.style.color = 'red'
     } else {
-        element.style.color = "green"
+        header.style.color = 'black'
     }
-}
+})
 
 /***** Deliverable 2 *****/
+const like_button = document.querySelector('button.like-button')
+
+like_button.addEventListener('click', function (e) {
+    console.log('likedddd')
+    traveler.likes += 1
+    likes.textContent = `${traveler.likes} Likes`
+
+})
 
 /***** Deliverable 3 *****/
+
+const form = document.querySelector('form#new-animal-sighting-form')
+form.addEventListener("submit", function (e) {
+    e.preventDefault()
+
+    const input1 = e.target[0].value
+    const input2 = e.target[1].value
+    const input3 = e.target[2].value
+    const input4 = e.target[3].value
+
+    const newAnimalObject = {
+        Species: input1,
+        VideoLink: input2,
+        PhotoLink: input3,
+        Description: input4,
+        likes: 0,
+        id: traveler.animalSightings[traveler.animalSightings.length - 1].id + 1
+
+    }
+    traveler.animalSightings.push(newAnimalObject)
+    renderAnimalSightingPost(newAnimalObject)
+    form.reset()
+})
```

Comment:
good use of `querySelector` and specifying that you were looking for a form with an id of `new-animal-sighting-form`! 

a couple of things to keep in mind for future reference:
1. using more descriptive variable naming. for example, instead of `input1` use `speciesInput`. this will help the dev that's working on your codebase later understand what value you're saving to the variable.  
2. inside of the newAnimalObject, double check that the key (Species, VideoLink, PhotoLink, and Description) matches the keys that the function`renderAnimalSightingPost` are pulling out to use. in this case, the function was looking for the lowercase version, so: 
` let newAnimalObject = {
        species: input1,
        link: input2,
        photo: input3,
        description: input4,
        id: traveler.animalSightings[traveler.animalSightings.length - 1].id + 1
    }`